### PR TITLE
[7.5.x] AF-1079: Empty 'Validation errors' popup in Decision/Business Central

### PR DESCRIPTION
**Pull request by Rikkola**
(cherry picked from commit ba21e5c92e74bee811814b29755d4407ba62debf)

https://issues.jboss.org/browse/AF-1079

https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/728
https://github.com/kiegroup/appformer/pull/271
https://github.com/kiegroup/kie-wb-common/pull/1561
https://github.com/kiegroup/drools-wb/pull/839

### CODE REVIEW

**Comment by Rikkola** [comment]
Looks like random tests. Jenkins please retest this

File: kie-wb-common-widgets/kie-wb-metadata-widget/src/main/java/org/kie/workbench/common/widgets/metadata/client/KieEditor.java [hunk 1]
```diff
@@ -379,6 +380,11 @@ protected void save(final String commitMessage) {
 
     }
 
+    protected void notifyValidationSuccess() {
```
Comment (karreiro) [inline kie-wb-common-widgets/kie-wb-metadata-widget/src/main/java/org/kie/workbench/common/widgets/metadata/client/KieEditor.java]: Ohh! One more thing. Where is this method being called?

File: kie-wb-common-widgets/kie-wb-common-ui/src/main/java/org/kie/workbench/common/widgets/client/callbacks/AssetValidatedCallback.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            validationPopup.showMessages(validationMessages);
+        }
+        if (validationFinishedCommand != null) {
+            validationFinishedCommand.execute();
+        }
+    }
+
+    private void notifyValidationSuccess() {
+        // the null check is due to tests that are not able to inject Event instance
+        if (notificationEvent != null) {
```
[the remark this answers is not in the document]
Comment (Rikkola) [reply kie-wb-common-widgets/kie-wb-common-ui/src/main/java/org/kie/workbench/common/widgets/client/callbacks/AssetValidatedCallback.java]: I think this was just to make mocking this class easier when you don't want to do a long setup with all the GWT classes. Due to Constants and Event. Source Mock might do the same thing.

File: kie-wb-common-widgets/kie-wb-metadata-widget/src/main/java/org/kie/workbench/common/widgets/metadata/client/KieEditor.java (hunk 1 above)
Comment (Rikkola) [reply kie-wb-common-widgets/kie-wb-metadata-widget/src/main/java/org/kie/workbench/common/widgets/metadata/client/KieEditor.java]: Looks like left over code.

**Pull request merged**